_docs/technical/2017-01-04-updater.md

```diff
@@ -93,14 +99,17 @@ As nodes deserialize
 they modify mempool as implemented
 [here](https://github.com/input-output-hk/cardano-sl/blob/22360aa45e5dd82d0c87872d8530217fc3d08f4a/src/Pos/Update/MemState/Functions.hs#L40).
 
+[//]: TODO: what to say about votes?
```

Comment:
What do you think would be useful to have regarding votes? @gromakovsky you might have some ideas.